finetune_adapter.py

```diff
@@ -4,6 +4,10 @@
 LLaMA-Adapter: Efficient Fine-tuning of Language Models with Zero-init Attention
 https://arxiv.org/abs/2303.16199
 
+This script uses DeepSpeed Zero-2 to train efficiently on 8 A100 GPUs within 1 hour as done in the original paper.
+If you have fewer GPUs, you can remove the `DeepSpeedStrategy` from Fabric and set `devices = 1`.
```

Comment:
The script already sets strategy to `None` if devices are <2, so probably this comment should be changed to reflect that